### PR TITLE
Add some options to the chromebook-setup.sh script

### PR DESCRIPTION
**Pull request by martinezjavier**
This pull-request makes a few enhancements to the chromebook-setup.sh script, by adding a some options and allowing to specify the architecture to be built by using the `ARCH` environment variable.

Javier Martinez Canillas (5):
      chromebook-setup: Allow to specify the arch exporting an ARCH variable
      chromebook-setup: Add --help option
      chromebook-setup: Add option to set path to linux git repository
      chromebook-setup: Add option to set a linux distro to be supported
      fragments: Add a Fedora Kconfig fragment

### CODE REVIEW

**Review by ericcurtin — APPROVED** [review]

File: chromebook-setup.sh [hunk 1]
```diff
@@ -142,18 +159,29 @@ or to do the same to use NFS for the root filesystem:
...
 while true; do
     case "$1" in
+	--help|-h)
+	    print_usage_exit
+	    ;;
+	--distro)
+	    CB_DISTRO="$2"
+	    shift 2
+	    ;;
+	--repository)
```
Comment (eballetbo) [inline chromebook-setup.sh]: Should we use --kernel instead of --repository? Repository looks a bit ambiguous to me. What do you think?

Comment (martinezjavier) [reply chromebook-setup.sh]: I actually called this kernel at the beginning but then decided to go with repo. I'll change it.

**Review by eballetbo — APPROVED** [review]
LGTM

**Comment by ericcurtin** [comment]
Be tempting to stick

```
if [ "$ARCH" == "aarch64" ]; then
  ARCH="arm64"
fi
```

somewhere at the top of this script somewhere just to cover that also.

**Comment by ericcurtin** [comment]
Like we seem to do here:

`[ "$ARCH" = "amd64" ] && ARCH="x86_64"`

**Comment by martinezjavier** [comment]
> Be tempting to stick
> 
> ```
> if [ "$ARCH" == "aarch64" ]; then
>   ARCH="arm64"
> fi
> ```
> 
> somewhere at the top of this script somewhere just to cover that also.

Yeah, I think that makes sense. Feel free to propose a PR.

**Comment by ericcurtin** [comment]
https://github.com/eballetbo/chromebooks/pull/11

Done

**Pull request merged**